src/AssetManager.php

```diff
@@ -362,13 +336,13 @@ private function registerAssetBundle(string $name, ?int $position = null): Asset
             if ($pos === null) {
                 $bundle->jsOptions['position'] = $pos = $position;
             } elseif ($pos > $position) {
-                throw new \RuntimeException(
+                throw new RuntimeException(
                     "An asset bundle that depends on '$name' has a higher javascript file " .
                     "position configured than '$name'."
                 );
             }
 
-            // update position for all dependencies
+            /** update position for all dependencies */
```

Comment:
It is inline comment and for inline comments there's no need to use phpdoc.

Reply:
Fixed.